### PR TITLE
Counter and others

### PR DESCRIPTION
**Pull request by jmaguero**
I've added an animated counter section with the number of members we have now.
Also I've updated the tailwind version.

### CODE REVIEW

File: pages/Counter.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      if (frame === totalFrames) {
+        clearInterval(counter);
+      }
+    }, frameDuration);
+  }, []);
+
+  return Math.floor(count);
+};
+
+const Counter = () => (
```
Comment (Woile) [inline pages/Counter.js:28]: I don't think this is `Counter`.
It is more like `Information` component.

File: pages/Counter.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+
+const easeOutQuad = t => t * (2 - t);
+const frameDuration = 1000 / 60;
+const members = 24;
+
+const CountUpAnimation = ({ children, duration = 2000 }) => {
```
Comment (Woile) [inline pages/Counter.js:7]: This should be `Counter`

File: tailwind.config.js [hunk 3]
```diff
@@ -6,6 +6,9 @@ module.exports = {
         // Complex site-specific row configuration
         layout: "1fr",
       },
+      colors: {
+        orange: '#ed8936',
```
Comment (Woile) [inline tailwind.config.js:10]: where is this being used?

File: pages/_app.js [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import '../styles/globals.css'
+import 'tailwindcss/tailwind.css'
```
Comment (Woile) [inline pages/_app.js:2]: Why are we introducing this here if we have them in the `css`?
https://github.com/Woile/bladers/blob/master/styles/globals.css

File: pages/Counter.js (hunk 1 above)
Comment (jmaguero) [reply pages/Counter.js:28]: Ok, I'll change it.

File: tailwind.config.js (hunk 3 above)
Comment (jmaguero) [reply tailwind.config.js:10]: It was used but it it not used anymore. I erased. :)

File: pages/_app.js (hunk 4 above)
Comment (jmaguero) [reply pages/_app.js:2]: Because that is what the Tailwind's documentation says for nextjs

File: components/Counter.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        clearInterval(counter);
+      }
+    }, frameDuration);
+  }, []);
+
+  return Math.floor(count);
+};
+
+
+export class Counter extends Component {
```
Comment (Woile) [inline components/Counter.js:30]: Do we need this class? It's empty!

Comment (jmaguero) [reply components/Counter.js:30]: Yes, it's the members counter' animation...

Comment (Woile) [reply components/Counter.js:30]: This class is not needed, is adding unnecessary overhead.

**Pull request merged**